starboard/shared/posix/impl/file_flush.h

```diff
@@ -36,7 +37,18 @@ bool FileFlush(SbFile file) {
     return false;
   }
 
-  return !HANDLE_EINTR(fsync(file->descriptor));
+  // Open a file stream by descriptor
+  FILE* stream = fdopen(file->descriptor, "a");
+  if (stream != NULL) {
+    // Flush only down to kernel buffers level
+    fflush(stream);
+    // Reassign stream to "/dev/null" without closing descriptor
+    freopen(NULL, "a", stream);
+    return true;
```

Comment:
As far as I know, fflush will only flush the user space buffers of the "FILE" stream and not call fsync or fdatasync on the underlying fd, so this early return means that fsync or fdatasync will not be called on the file.